composer.json

```diff
@@ -82,7 +82,9 @@
         "doctrine/dbal": "^2.6|^3.0",
         "filp/whoops": "^2.8",
         "guzzlehttp/guzzle": "^7.2",
-        "league/flysystem-cached-adapter": "^1.0",
+        "league/flysystem-aws-s3-v3": "^2.0",
+        "league/flysystem-ftp": "^2.0.0-RC1",
```

Comment:
Why not `^2.0`?

Reply:
See https://github.com/laravel/framework/pull/33612#discussion_r465021969

Reply:
That comment no longer applies now that 2.0.0 is tagged.

Reply:
ah correct!